### PR TITLE
Added "Create package" button in "+Offline" #567

### PR DESCRIPTION
**Pull request by christian1020-94**
For #567.

### CODE REVIEW

File: src/zabapgit_forms.prog.abap [hunk 1]
```diff
@@ -119,6 +119,52 @@ FORM branch_popup TABLES   tt_fields TYPE ty_sval_tt
...
+  DATA: ls_package_data TYPE scompkdtln.
+
+  FIELD-SYMBOLS: <ls_fbranch> LIKE LINE OF tt_fields.
+
+  CLEAR cs_error.
+
+  IF pv_code = 'COD1'.
+    cv_show_popup = abap_true.
+
+    CALL FUNCTION 'FUNCTION_EXISTS'
```
Comment (larshp) [inline src/zabapgit_forms.prog.abap]: can we refactor the call to FUNCTION_EXISTS + PB_POPUP_PACKAGE_CREATE to a new static method in `lcl_popups`? the 20 lines already exist in this include.

Comment (christian1020-94) [inline src/zabapgit_forms.prog.abap]: Thanks for the suggestion. Done.

File: src/zabapgit_popups.prog.abap [hunk 2]
```diff
@@ -61,12 +61,12 @@ CLASS lcl_popups DEFINITION FINAL.
...
-                text_message          TYPE clike
-      RAISING   lcx_exception.
-
-
+        IMPORTING
+                  titlebar              TYPE clike
+                  text_message          TYPE clike
+        RAISING   lcx_exception,
+      popup_to_create_package
+        RETURNING VALUE(rs_package_data) TYPE scompkdtln.
```
Comment (larshp) [inline src/zabapgit_popups.prog.abap]: add `RAISING lcx_exception`

Comment (larshp) [inline src/zabapgit_popups.prog.abap]: add `RAISING lcx_exception`

File: src/zabapgit_forms.prog.abap [hunk 3]
```diff
@@ -119,6 +100,33 @@ FORM branch_popup TABLES   tt_fields TYPE ty_sval_tt
...
+  FIELD-SYMBOLS: <ls_fbranch> LIKE LINE OF tt_fields.
+
+  CLEAR cs_error.
+
+  IF pv_code = 'COD1'.
+    cv_show_popup = abap_true.
+
+    ls_package_data = lcl_popups=>popup_to_create_package( ).
+
+    lcl_sap_package=>create( ls_package_data ).
```
Comment (larshp) [inline src/zabapgit_forms.prog.abap]: if the user presses cancel in the popup, then `ls_package_data` is initial and no package should be created

Comment (christian1020-94) [inline src/zabapgit_forms.prog.abap]: You're right, give me a minute.

**Comment by larshp** [comment]
Thanks, merging

there is also a `lcx_cancel` exception that sometime is used, but for now it will work

**Comment by christian1020-94** [comment]
Thanks for the suggestions Lars.

**Pull request merged**